kvm-install-vm

```diff
@@ -714,6 +790,17 @@ function set_defaults ()
 
 function set_custom_defaults ()
 {
+    # Advanced hypervisor options. Override in ~/.kivrc if needed.
```

Comment:
I think it would be better to set these defaults along with the rest in `set_defaults ()`.

Reply:
Thank you for the feedback. I was unsure if it would be better to have them in set_defaults() or set_custom_defaults(). I will move them back to set_defaults().

Reply:
Pathset set has been updated (git commit --amend; git push --force)